tck/README.adoc

```diff
@@ -101,3 +112,28 @@ For this reason, the rows of the expected results table are to be considered a s
 In order to implement the Cypher TCK, you will have to retrieve the full suite of TCK feature files, hosted at this GitHub repository, https://github.com/opencypher/openCypher/tree/master/tck/features[in this subdirectory].
 
 // TODO: Mention Cucumber ?
+
+[[errors]]
+=== Cypher errors
+
+The `Then` step used to specify expected errors from running a given invalid query follows this schematic setup:
+
+ Then a TYPE should be raised at TIME: DETAIL
+
```

Comment:
I don't think `TIME` is the best term here - I keep interpreting it as "12:45" and co. So, I suggest a less loaded term... maybe something like `QUERY_STAGE`, `EXECUTION_STAGE`, `ENVIRONMENT` and so on (I am a lot less keen on the latter term than the first two, but have added it in case it inspires something better).